.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,159 @@
+
+    Returns:
+        bool: True if eslint-disable statement is found, False otherwise.
+    """
+    # Initialize key variables
+    eslint_disable_pattern = re.compile(
+        r"\/\/.*eslint-disable.*|\/\*[\s\S]*?eslint-disable[\s\S]*?\*\/",
+        re.IGNORECASE | re.DOTALL,
+    )
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)